quotemachine/app.js

```diff
@@ -1,101 +1,41 @@
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.2<randomColorText <= 0.3) {
-        quote.innerHTML = "I don't need time, I need a deadline."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="blue"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.3<randomColorText <= 0.4) {
-        quote.innerHTML = "Dreams don't have timelines, deadlines, and aren't always in straight lines."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="grey"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.4<randomColorText <= 0.5) {
-        quote.innerHTML = "Normally I miss deadlines like a storm trooper misses Jedi."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="orange"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.5<randomColorText <= 0.6) {
-        quote.innerHTML = "There's nothing an artist needs more - even more than excellent tools and stamina - than a deadline."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="brown"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.6<randomColorText <= 0.7) {
-        quote.innerHTML = "There’s no getting out alive, but you hope to avoid a deadline."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="DarkSlateBlue"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.7<randomColorText <= 0.8) {
-        quote.innerHTML = "Respect your deadline like it’s a field marshal."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="Fuchsia"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.8<randomColorText <= 0.9) {
-        quote.innerHTML = "Deadline-induced quality issues: a problem all over the galaxy."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="MediumSlateBlue"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-    else if (0.9<randomColorText <= 1) {
-        quote.innerHTML = "I do however like to tell people, especially regarding writing and deadlines: Don't be perfect, just be done."
-        
-        for(let i=0;i<randomColor.length;i++){
-            randomColor[i].style.backgroundColor="Salmon"
-        }
-        
-        document.body.appendChild(quote.innerHTML);
-    }
-})
 
 
+
+const newQuoteButton = document.getElementById("new-quote");
+let quote = document.getElementById("quote-text");
+const randomColor = document.getElementsByClassName("random-color");
```

Comment:
A name like this can be very confusing. From looking at your code I understand that this constant should be holding an array of html elements with "random-color" class. 
When choosing a name for a constant, always try to find a name that would make the person reading it understand what value it holds just by reading the name itself. 
```suggestion
const elementThatChangeColor = document.getElementsByClassName("random-color");
```